### ISSUE TITLE
Crash on OS X when I will set a filter

### ISSUE THREAD

**Issue by logisch** [mltframework/shotcut#182]
When I will set a filter and select them (fade-in or others), close Shortcut. 
System: Mac OS X 10.11.2 El Capitan
The error can be reproduced.

**Comment by ddennedy** [comment]
Unable to reproduce. Needs more details or a patch because if I cannot reproduce, I cannot fix.

**Comment by ddennedy** [comment]
yeah, I'm just going to close this as I am unable to reproduce it plus not specific about filters and no steps and no version = invalid

**Comment by logisch** [comment]
I make following steps:
Open a file
Cut a part 
New filter for fade-in or other filter
Application crash and close

It does not matter if were already taken steps or a new file is opened. When I use a filter I have a crash.

 Modellname:	MacBook Pro
  Modell-Identifizierung:	MacBookPro8,1
  Prozessortyp:	Intel Core i7
  Prozessorgeschwindigkeit:	2.7 GHz
  Anzahl der Prozessoren:	1
  Gesamtanzahl der Kerne:	2
  L2-Cache (pro Kern):	256 KB
  L3-Cache:	4 MB
  Speicher:	8 GB
  Boot-ROM-Version:	MBP81.0047.B2C

I hope, it help you.

**Comment by ddennedy** [comment]
I am sorry, but "cut a part" is not well-defined (does that involve the playlist or timeline or neither?), and I do not know if you are using the latest version 16.01 of Shotcut with GPU processing disabled. I have tested a few different approaches to "cut," adding both audio and video fade-in filters, and switching between the filters. I tested Shotcut 16.01 with GPU processing and JACK disabled on a MacBookPro11,1 running OS X 10.11.2.
